src/components/custom/dashboard/shared/chat/RealTimeChatInterface.jsx

```
@@ -0,0 +1,721 @@
+
+
+  return (
+    <div className="flex h-full flex-col">
+      {/* Chat header */}
+      <div className="flex items-center justify-between border-b border-[#333] p-4">
+        <div className="flex items-center">
+          <button
+            onClick={onClose}
+            className="mr-3 rounded p-1 hover:bg-[#333] lg:hidden"
+          >
+            ←
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)